charts/kubescape-operator/templates/configs/cloud-secret.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ .Values.global.cloudSecret }}
+  namespace: {{ .Values.ksNamespace }}
+  labels:
```

Comment:
We should have a hard-coded label here so we will identify this secret based on the label and not the secret name